.github/workflows/R-CMD-check.yaml

```diff
@@ -33,15 +33,15 @@ jobs:
           - {os: macOS-latest,   r: 'release'}
           - {os: windows-latest, r: 'release'}
           - {os: ubuntu-latest,  r: 'devel', http-user-agent: 'release'}
-          - {os: ubuntu-20.04,   r: 'release'}
```

Comment:
Question: maybe it is better just to jump to more recent version (24.04)?

Reply:
We have ubuntu-latest above with "R devel", so this one theoretically should check something more stable. I would also leave it as is to run R CMD check on something more mature